app/src/main/java/com/wizeline/academy/animations/ui/more_details/MoreDetailsFragment.kt

```diff
@@ -29,6 +38,17 @@ class MoreDetailsFragment : Fragment() {
     ): View {
         _binding = MoreDetailsFragmentBinding.inflate(inflater, container, false)
         binding.ivImageDetailLarge.loadImage(args.imageId)
+
+        val title = binding.tvTitle;
```

Comment:
does not need to store variables locally if you have access to them using binding.tvTitle everywhere